pgxn/neon/pagestore_smgr.c

```diff
@@ -1726,7 +1726,7 @@ neon_extend(SMgrRelation reln, ForkNumber forkNum, BlockNumber blkno,
 		if (current_size >= ((uint64) max_cluster_size) * 1024 * 1024)
 			ereport(ERROR,
 					(errcode(ERRCODE_DISK_FULL),
-					 errmsg("could not extend file because project size limit (%d MB) has been exceeded",
+					 errmsg(NEON_TAG "could not extend file because project size limit (%d MB) has been exceeded",
```

Comment:
This message is seen by users regularly, when they hit the limit. I wonder if we should leave out the neon tag from this one.

Maybe leave out the "internally by neon.max_cluster_size" part too, that can be confusing to users. But that can be done separately.

Reply:
Fixed (remove [NEON_SMGR] prefix from this message